data/json/items/armor/boots.json

```diff
@@ -1032,5 +1032,45 @@
     "material_thickness": 2,
     "environmental_protection": 1,
     "flags": [ "VARSIZE", "WATER_FRIENDLY", "SKINTIGHT" ]
+  },
+  {
+    "id": "roller_shoes_off",
+    "type": "ARMOR",
+    "name": "pair of heelys (off)",
+    "name_plural": "pairs of heelys (off)",
+    "description": "A pair of sneakers with wheels, currently the wheels are hidden.",
+    "weight": 840,
+    "volume": 6,
+    "price": 10000,
+    "material": [ "cotton", "leather" ],
+    "symbol": "[",
+    "color": "light_blue",
+    "covers": [ "FEET" ],
+    "coverage": 80,
+    "encumbrance": 4,
+    "warmth": 20,
+    "material_thickness": 2,
+    "use_action": { "type": "transform", "target": "roller_shoes_on", "msg": "You pop the wheels out.", "moves": 500 },
+    "flags": [ "VARSIZE" ]
+  },
+  {
+    "id": "roller_shoes_on",
+    "type": "ARMOR",
+    "name": "pair of heelys (on)",
+    "name_plural": "pairs of heelys (on)",
+    "description": "A pair of sneakers with wheels, currently the wheels are out.",
+    "weight": 840,
+    "volume": 6,
+    "price": 10000,
+    "material": [ "cotton", "leather" ],
+    "symbol": "[",
+    "color": "light_blue",
+    "covers": [ "FEET" ],
+    "coverage": 80,
+    "encumbrance": 4,
+    "warmth": 20,
+    "material_thickness": 2,
+    "use_action": { "type": "transform", "target": "roller_shoes_off", "msg": "You pop the wheels back in.", "moves": 500 },
+    "flags": [ "VARSIZE", "HEELIES", "ROLLER_ONE", "REQUIRES_BALANCE" ]
```

Comment:
Why do you need two flags - HEELIES and ROLLER_ONE? I'm pretty sure one is enough.

Reply:
Fair, this was set up in such a way that roller_one and heelies would be separate, as they do separate things which is 

Roller One: it modifies the shoe itself, changing the speed

Heelies: its just a stand-alone flag that modifies requires_balance so that any item that has it changes it so the balance has something added to it to make the item less unstable. 

I figured I would keep them separate in case someone might need the use of heelies but not roller_one and vice versa, but if bloat is a worry, then I can probably remove heelies and change the flag in requires_balance from heelies to roller_one.

Reply:
Yeah, please leave only the necessary stuff.
Also, please add a documentation for your new flag in json_flags.md.

Reply:
I thought I did? Strange, I guess it didn't save, let me remove heelies flag and then re-add the flag to json_flags

Reply:
Alright, I removed HEELIES flag, as well as not only adding ROLLER_ONE to json_flags, but ROLLER_QUAD and ROLLER_INLINE as well, as for some reason they are missing.